pontoon/sync/core.py

```diff
@@ -53,7 +53,13 @@ def wrapper(func):
         @shared_task(bind=True, **celery_args)
         @wraps(func)
         def wrapped_func(self, *args, **kwargs):
-            lock_name = "serial_task.sync[{}]".format(lock_key.format(*args, **kwargs))
+            if func.__name__ == "pretranslate":
+                lock_name = "task.pretranslate[{}]".format(lock_key.format(args[0].pk))
+            else:
+                lock_name = "serial_task.sync[{}]".format(
+                    lock_key.format(*args, **kwargs)
+                )
```

Comment:
We should generalize this, and that shouldn't be too hard now that we only have 1 sync task:
https://github.com/mozilla/pontoon/commit/dd5e48051abb2f02f4d3591c1566ca7f9ab9f1ec#diff-2be3ba079451045c9badba06c960b1d0